src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 230;
```

Comment:
The return value '230' is a hardcoded literal. Use a constant for the daily cost and calculate the total cost based on the number of days, applying discounts when appropriate.